jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
+  thebe: true
+
+
+#######################################################################################
+# Execution settings
+execute:
+  execute_notebooks           : cache
+
+#######################################################################################
+# HTML-specific settings
+html:
+  home_page_in_navbar         : false
+
+
+# #######################################################################################
+# Interact link settings #
+notebook_interface            : "notebook"
+
+#######################################################################################
+# Launch button settings
+#repository:
+#  url                         : https://github.com/
+
+#  use_repository_button     : true  # Whether to add a link to your repository button
+#  use_issues_button         : true  # Whether to add an "open an issue" button
```

Comment:
We need the open issue. That is how we interact with users to request new notebooks.

Reply:
Okay. Have turned this on.

Reply:
Are there other features here, like star the notebook or give a +1 or something like that to get more feedback from users?

Reply:
There isn't any option of upvoting the notebook for now but might get it in the future builds of Jupyter{Book}.